src/rf/apps/home/views.py

```diff
@@ -199,13 +202,37 @@ def _get_layer_models(request, crit=None):
     if crit:
         qs = qs.filter(crit)
 
-    layers = LayerFilter(request.GET, queryset=qs)
-    return layers
+    filtered_layers = LayerFilter(request.GET, queryset=qs)
+    paginator = Paginator(filtered_layers, RESULTS_PER_PAGE)
+
+    page = request.GET.get('page')
+    try:
+        layers = paginator.page(page)
+    except PageNotAnInteger:
+        # If page is not an integer, deliver first page.
+        page = 1
+        layers = paginator.page(page)
+    except EmptyPage:
+        # If page is out of range, deliver last page of results.
+        page = paginator.num_pages
+        layers = paginator.page(page)
+
+    return {
+        'layers': layers,
+        'pages': paginator.num_pages,
+        'current_page': page
+    }
 
 
 def _get_layers(request, crit=None):
     """
     Return list of JSON serializable layer models.
     """
-    models = _get_layer_models(request, crit)
-    return [m.to_json() for m in models]
+    results = _get_layer_models(request, crit)
+    models = [m.to_json() for m in results['layers']]
+
+    return {
+        'layers': models,
+        'pages': results['pages'],
+        'current_page': results['current_page']
```

Comment:
This felt more in line with the front end possibilities.

Often we see pagination controls that look like this:

```
1, 2, 3, 4, 5, ... last
```
and those often change based on where you are in the pager:

```
first ... 98, 99, 100
```

So having a next/prev URL is not as helpful as actually knowing how many pages there are and which one is current.

I'm not against adding a prev/next URL as well but I think we should keep the page metadata.
```